ui/apps/platform/src/services/AuthService/AuthService.ts

```diff
@@ -183,6 +172,29 @@ export function deleteAuthProviders(authProviderIds) {
     return Promise.all(authProviderIds.map((id) => deleteAuthProvider(id)));
 }
 
+function convertAuthProviderClaimMappingsToArray(provider: AuthProvider): AuthProvider {
+    if (!provider.claimMappings) {
+        return provider;
+    }
+    const mappingAsArray = Object.entries(provider.claimMappings).sort((a, b) =>
+        a[0].localeCompare(b[0])
+    );
+    const newProvider = { ...provider };
+    newProvider.claimMappings = mappingAsArray;
+    return newProvider;
+}
+
+function convertAuthProviderClaimMappingsToObject(authProvider: AuthProvider): AuthProvider {
+    if (!authProvider.claimMappings) {
+        return authProvider;
+    }
+    const newProvider = { ...authProvider };
+    newProvider.claimMappings = Object.fromEntries(
+        authProvider.claimMappings as [string, string][]
```

Comment:
An observation, not a change request: with `!Array.isArray(authProvider.claimMappings)` instead of `!authProvider.claimMappings` in `if` statement for early return, `as` might not be needed.